### PR TITLE
Callhome: support the News service

### PR DESCRIPTION
**Pull request by psiinon**
Fixes https://github.com/zaproxy/zaproxy/issues/6016

Signed-off-by: Simon Bennetts <psiinon@gmail.com>

### CODE REVIEW

File: addOns/callhome/src/main/java/org/zaproxy/addon/callhome/InvalidServiceUrlException.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+ * limitations under the License.
+ */
+package org.zaproxy.addon.callhome;
+
+public class InvalidServiceUrlException extends Exception {
+
+    private static final long serialVersionUID = 1L;
+
+    public InvalidServiceUrlException(String url) {
+        super(url);
```
Comment (kingthorin) [inline addOns/callhome/src/main/java/org/zaproxy/addon/callhome/InvalidServiceUrlException.java:27]: Won't this mean the whole exception message is just the URL?

File: addOns/callhome/src/main/javahelp/org/zaproxy/addon/callhome/resources/help/contents/callhome.html [hunk 2]
```diff
@@ -15,5 +15,8 @@ <H2>Check For Updates</H2>
 The Check for Updates call is used to find out the latest information about ZAP and the published add-ons. 
 It is a mandatory add-on from ZAP 2.12.0 as this feature is used by the ZAP core.
 
+<H2>News</H2>
+The News call is used to find out the latest new item about ZAP - this is displayed in the ZAP desktop. Quick Start panel
```
Comment (kingthorin) [inline addOns/callhome/src/main/javahelp/org/zaproxy/addon/callhome/resources/help/contents/callhome.html:19]: Could use a period.
Also displayed in launched browsers right?

**Review by kingthorin — APPROVED** [review]

File: addOns/callhome/src/main/javahelp/org/zaproxy/addon/callhome/resources/help/contents/callhome.html (hunk 2 above)
Comment (psiinon) [reply addOns/callhome/src/main/javahelp/org/zaproxy/addon/callhome/resources/help/contents/callhome.html:19]: Uh, added the text at the wrong place. Pretty sure the new is not displayed in the launched browsers, but clicking on the news button does launch a browser..

File: addOns/callhome/src/main/java/org/zaproxy/addon/callhome/InvalidServiceUrlException.java (hunk 1 above)
Comment (psiinon) [reply addOns/callhome/src/main/java/org/zaproxy/addon/callhome/InvalidServiceUrlException.java:27]: I just ripped off the previous AutoUpdate exception :P

File: addOns/callhome/src/main/javahelp/org/zaproxy/addon/callhome/resources/help/contents/callhome.html (hunk 2 above)
Comment (kingthorin) [reply addOns/callhome/src/main/javahelp/org/zaproxy/addon/callhome/resources/help/contents/callhome.html:19]: You're right, Browser Launch displays the manual explore message.

Comment (psiinon) [reply addOns/callhome/src/main/javahelp/org/zaproxy/addon/callhome/resources/help/contents/callhome.html]: Fixed the fullstop :)

**Review by kingthorin — APPROVED** [review]

**Review by kingthorin — APPROVED** [review]

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Pull request merged**